controllers/storagecluster/provider_server.go

```diff
@@ -454,6 +454,8 @@ func getOnboardingJobObject(instance *ocsv1.StorageCluster) *batchv1.Job {
 			Namespace: instance.Namespace,
 		},
 		Spec: batchv1.JobSpec{
+			// Eligible to delete automatically when job finishes
+			TTLSecondsAfterFinished: util.ToPointer[int32](0),
```

Comment:
@jarrpa pointed out that k8s vendor already provides a library with similar functionality to `utils.ToPointer`:  https://pkg.go.dev/k8s.io/utils/ptr 

@rchikatw Can we please use that implementation instead and remove our util from the code base?